lua/visual-whitespace.lua

```diff
@@ -203,13 +203,9 @@ M.setup = function(user_cfg)
     ['\n'] = CFG['nl_char'],
     ['\r'] = CFG['cr_char']
   }
+  vim.print(CFG['highlight'])
```

Comment:
This PR is good to go except this print statement!

Reply:
Fixed it!

Reply:
Sorry 🤦🏼‍♂️

Reply:
Nothing to be sorry about 👍🏻 I do that kind of stuff all the time haha